js/libs/keycloak-admin-client/src/defs/requiredActionProviderRepresentation.ts

```diff
@@ -7,7 +7,7 @@ export enum RequiredActionAlias {
   UPDATE_PROFILE = "UPDATE_PROFILE",
   CONFIGURE_TOTP = "CONFIGURE_TOTP",
   UPDATE_PASSWORD = "UPDATE_PASSWORD",
-  terms_and_conditions = "terms_and_conditions",
+  TERMS_AND_CONDITIONS = "TERMS_AND_CONDITIONS",
```

Comment:
This file is present in `release/21.0` branches in both `keycloak/keycloak` and [`keycloak/keycloak-ui`](https://github.com/keycloak/keycloak-ui/pull/4582/files#diff-68b48072a43f87c0b010a58390a33827e3c3beba5c4fa91f6a99e7d8fb8f6c61R10)

Not sure which one is actually used during release, but I changed it in both places. Please let me know if I should do something else.

Reply:
@jonkoops I understand why is the PR to `keycloak-ui` needed, I am not questioning that. 

What I am pointing out in this comment is, that this module is present in both repositories for branch `release/21.0`
https://github.com/keycloak/keycloak/tree/release/21.0/js/libs/keycloak-admin-client
https://github.com/keycloak/keycloak-ui/tree/release/21.0/libs/keycloak-admin-client

Reply:
@mhajas when we released 21 we had partially started moving the UI bits into the main repository, but looking at the [workflows](https://github.com/keycloak-rel/keycloak-rel/blob/release/21.0/.github/workflows/x-keycloak-nodejs-admin.yml) we are still releasing admin client from the keycloak-nodejs-admin-client repository for 21.